docs/meshstack.azure.landing-zones.md

```diff
@@ -91,7 +91,10 @@ This flag allows you to use a User Assigned Managed Identity (UAMI) instead of t
 
 Currently we only recommend to use this flag if you want to create resources that require permissions outside of the meshcloud managed subscription. A good example is if you want to connect a meshcloud managed subscription to a central log workspace. In order to do this, you require permissions in both the meshcloud managed subscription and the subscription your central Log Workspace App resides in.
 
-**Please note**: meshcloud automatically handles permissions in the subscription the Blueprint is assigned to, but permissions outside of that cannot be handled and have to be setup by yourself.
+>**Please note**: meshStack can not handle permissions outside of the Subscriptions it manages. You need to make sure the Managed Identity has the correct permissions set. You must also give the replicator service principal the following permission on the target resource the Managed Identity lives in (e.g. the Subscription of the Managed Identitiy).
```

Comment:
what are the correct permissions for the managed identity?

Reply:
Could this be added to [https://docs.dev.meshcloud.io/docs/meshstack.azure.index.html#azure-rbac-permissions](https://docs.dev.meshcloud.io/docs/meshstack.azure.index.html#azure-rbac-permissions) or is this to specific to be added in the general setup of the replicator SPN?

Reply:
depends on what you want to do with the principal e.g. all what is required to assign your blueprint. But I can clarify on that.

Reply:
You need to add the permissions the Blueprint requires to do its work. I clarify on that.